yarn.lock

```diff
@@ -494,7 +494,7 @@
   resolved "https://registry.yarnpkg.com/@types/pino-std-serializers/-/pino-std-serializers-2.4.0.tgz#8cad99175cb79c2448f7455a2d32fb3fde29579c"
   integrity sha512-eAdu+NW1IkCdmp85SnhyKha+OOREQMT9lXaoICQxa7bhSauRiLzu3WSNt9Mf2piuJvWeXF/G0hGWHr63xNpIRA==
 
-"@types/pino@^5.8.10":
+"@types/pino@^5.14.0", "@types/pino@^5.8.10":
```

Comment:
We should bump this across the different services  some time soon so we don't have multiple version dependencies.

Reply:
Agreed - but this is one of the reasons we have services - they are independent :smile: